core/src/main/java/com/sequenceiq/cloudbreak/service/upgrade/image/BlueprintUpgradeOptionValidator.java

```diff
@@ -71,6 +75,6 @@ private BlueprintValidationResult isEnabledForCustomBlueprint(Blueprint blueprin
 
     private BlueprintValidationResult createResult(boolean result, BlueprintUpgradeOption upgradeOption) {
```

Comment:
minor: BlueprintUpgradeOption upgradeOption seems to be unused here.

Reply:
good catch, fixed.